packages/core/src/external/carequality/ihe-gateway-v2/outbound/xca/send/dq-requests.ts

```diff
@@ -21,14 +21,15 @@ export async function sendSignedDqRequest({
   patientId,
   cxId,
   index,
+  trustedKeyStore,
 }: {
   request: SignedDqRequest;
   samlCertsAndKeys: SamlCertsAndKeys;
   patientId: string;
   cxId: string;
   index: number;
+  trustedKeyStore: string;
 }): Promise<DQSamlClientResponse> {
-  const trustedKeyStore = await getTrustedKeyStore();
```

Comment:
My suggestion was to have `const trustedKeyStore = await getTrustedKeyStore();` on this file as a global const, outside of the function, not as a parameter.

It doesn't seem like it make sense to clients of this function/module to have to pass the trusted keystore, right? This seem an implementation detail.